src/pt/readmangas/src/eu/kanade/tachiyomi/extension/pt/readmangas/ReadMangas.kt

```diff
@@ -247,13 +247,16 @@ class ReadMangas() : HttpSource() {
 
     override fun pageListParse(response: Response): List<Page> {
         val document = response.asJsoup()
-        val script = document.select("script").map { it.data() }
-            .firstOrNull { IMAGE_URL_REGEX.containsMatchIn(it) }
-            ?: return emptyList()
+        val scripts = document.select("script").map { it.data() }
+        val pages = mutableListOf<Page>()
```

Comment:
Avoid using mutable objects if possible. Since you are just return it here you should be able to use an immutable list.

Reply:
Fix it. I used an immutable list.